python_modules/dagster/dagster/_generate/templates/CODE_LOCATION_NAME_PLACEHOLDER/CODE_LOCATION_NAME_PLACEHOLDER/__init__.py

```diff
@@ -2,4 +2,8 @@
 
 from . import assets
 
-defs = Definitions(assets=load_assets_from_modules([assets]))
+all_assets = load_assets_from_modules([assets])
+
+defs = Definitions(
+    assets=all_assets
```

Comment:
I think you need to add a comma at the end of this line or the auto-formatter will try to collapse it onto a single line.

Reply:
Odd, my precommit hook didn't kick in. updated and ran `make black` to be sure! ty!